command-snapshot.json

```diff
@@ -2,13 +2,37 @@
   {
     "command": "force:limits:api:display",
     "plugin": "@salesforce/plugin-limits",
-    "flags": ["apiversion", "json", "loglevel", "targetusername"],
```

Comment:
what happens if someone supplies currently valid flags for apiversion or loglevel ?

Reply:
@mshanemc No error is thrown and command finishes w/o error. I assume the same with loglevel

Reply:
sf does not support them natively and these commands cannot reasonably consume api version.